packages/client/__tests__/components/ResourceTable.spec.js

```diff
@@ -45,64 +51,69 @@ const tableDataBuilder = build('TableData', {
         scheduled_for: fake((f) => f.date.future()),
         updated_at: fake((f) => f.date.future()),
         description: fake((f) => f.lorem.words(12)),
-        title: fake((f) => f.lorem.word),
+        name: fake((f) => f.lorem.word),
         user_id: fake((f) => f.random.number()),
     },
 })
 
+const tableData = [
```

Comment:
Please replace this with `const tableData = Array.from({ length: 50 }).map(() => tableDataBuilder())`

Reply:
great idea, thanks